src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -75,7 +75,7 @@ public class WebScriptHostManager : ScriptHostManager
             config.IsSelfHost = webHostSettings.IsSelfHost;
 
             secretsRepositoryFactory = secretsRepositoryFactory ?? new DefaultSecretsRepositoryFactory();
-            var secretsRepository = secretsRepositoryFactory.Create(settingsManager, webHostSettings, config);
+            ISecretsRepository secretsRepository = secretsRepositoryFactory.Create(settingsManager, webHostSettings, config, loggerFactory.CreateLogger(ScriptConstants.LogCategoryMigration));
```

Comment:
Need to make sure this new category won't be filtered out. 

@brettsam can provide more information.

Reply:
Changed to "Host.Migration"